examples/mfa/mfacosigner/mfacosigner.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2024 OpenPubkey
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package mfacosigner
+
+import (
+	"crypto"
+	"crypto/rand"
+	"fmt"
+
+	"github.com/go-webauthn/webauthn/protocol"
+	"github.com/go-webauthn/webauthn/webauthn"
+	"github.com/lestrrat-go/jwx/v2/jwa"
+	"github.com/openpubkey/openpubkey/cosigner"
+)
+
+type UserKey struct {
+	Issuer string // ID Token issuer (iss)
+	Aud    string // ID Token audience (aud)
+	Sub    string // ID Token subject ID (sub)
+}
```

Comment:
Since we also define this struct in the cosigner folder, it seems like **either** this should import from there, or else this whole struct is just an example and we shouldn't have the definition in cosigner.

Reply:
UserKey should exist in `cosigner` and be imported from there since implementations should extend it. I deleted it from MFACosigner. That struct was unused. Should be fixed